### PR TITLE
Updated linux build instructions

### PR DESCRIPTION
**Pull request by tombogle**
@chrisvire These are some changes I made locally back in February based on a Slack conversation with you. Just saw that I never committed this back then.

- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1260)

### CODE REVIEW

**Comment by chrisvire** [comment]
@tombogle, someone doing regular Linux development with dotnet should probably review this.  How about @marksvc or @ermshiperete ?

**Comment by ermshiperete** [comment]
Thanks for the update @tombogle! I simplified the instructions in #1264.

**Comment by marksvc** [comment]
Thanks for your work on this, guys! Looks like @ermshiperete was able to further clean it up in #1264. 
Note that development instructions are also at https://github.com/sillsdev/libpalaso/wiki/Development . I'm not sure why it's in both places at present :-D

**Comment by ermshiperete** [comment]
> Note that development instructions are also at https://github.com/sillsdev/libpalaso/wiki/Development .

Thanks for pointing that out. I updated the links on the wiki and deleted that particular page.

**Comment by marksvc** [comment]
*[`README.md` line 44 at r2](https://reviewable.io/reviews/sillsdev/libpalaso/1260#-Nd15OpK56vtoQbwmGuZ:-Nd15OpL9pJoqNxWOUHj:b45sgt4) ([raw file](https://github.com/sillsdev/libpalaso/blob/a6a93ec9a85dc22528616408a7c0b2a5acb0bb9c/README.md#L44)):*
> ```Markdown
> ##### The rest of the stuff:
> - Add access to packages.microsoft.com repo for dotnet sdk:
> ```

It [looks like](https://packages.ubuntu.com/search?searchon=names&suite=all&section=all&keywords=mono-devel) package "mono-devel" is v 6.8.0.105 in both focal and jammy, and the instructions in the base below now include dotnet-sdk-6.0 and libcanberra-gtk-module. I wonder if there are not any jammy-specific needs left? I haven't tried it.

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ These libraries cannot currently be built on Linux releases later than Focal wit
 
   ```bash
   sudo apt update
-  sudo apt install libicu-dev dotnet-sdk-5.0 mono-complete
+  sudo apt install libicu-dev dotnet-sdk-6.0 mono-complete mono-devel msbuild libcanberra-gtk-module
```
Comment (ermshiperete) [inline README.md:60]: `mono-complete` depends on `mono-devel`, so it wouldn't be necessary to explicitly install it.

**Review by ermshiperete — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chrisvire) [reply README.md]: @tombogle, I guess remove "mono-complete" from that line?

**Comment by tombogle** [comment]
*[`README.md` line 60 at r2](https://reviewable.io/reviews/sillsdev/libpalaso/1260#gh-1309915781:-NfvAgWh4uIu3uI0n5Sq:bb6un1j) ([raw file](https://github.com/sillsdev/libpalaso/blob/e92f5fd32529595ea41ac603aa29263f980e069e/README.md#L60)):*
<details><summary><i>Previously, chrisvire (Chris Hubbard) wrote…</i></summary><blockquote>

@tombogle, I guess remove "mono-complete" from that line?
</blockquote></details>

Done

**Pull request merged**